docs/admin/network-plugins.md

```diff
@@ -43,3 +43,10 @@ The plugin requires a few things:
 * Kubelet must be run with the `--network-plugin=kubenet` argument to enable the plugin
 * Kubelet must also be run with the `--reconcile-cidr` argument to ensure the IP subnet assigned to the node by configuration or the controller-manager is propagated to the plugin
 * The node must be assigned an IP subnet through either the `--pod-cidr` kubelet command-line option or the `--allocate-node-cidrs=true --cluster-cidr=<cidr>` controller-manager command-line options.
+
+## Usage Summary
+
+* `--network-plugin=exec`: Must specify the `--network-plugin-dir` flag to the location of executables.
+* `--network-plugin=cni`: Specify the `--network-plugin-dir` flag to the location of CNI config file. Default to `/etc/cni/net.d`. Meanwhile, CNI binaries must be present in `/opt/cni/bin`.
+* `--network-plugin=kubenet`: Specify the `--network-plugin-dir` to path where the CNI `bridge` and `host-local` plugins are placed in addition to default path `/opt/cni/bin`.
```

Comment:
do we need both bridge and host-local plugins? if so please emphasize this. What happens if there's only one (what breaks)? 

Also --network-plugin-dir here specifies the location of plugins, but with --network-plugin=cni --network-plugin-dir has the location of CNI files? (according to your previous section)